mods/e2140/content/ed/infantry/a01/rules.yaml

```
@@ -0,0 +1,36 @@
+ed_infantry_a01:
+	Inherits@1: ^EdInfantry
+	Inherits@2: ^CoreArmedSelfInfantry
+	Tooltip:
+		Name: A01 Android
+	Valued:
+		Cost: 50
+	Buildable:
+		IconPalette:
+		Queue: Infantry.ED
+	Health:
+		HP: 28
+	RevealsShroud:
+		Range: 2c896
+	Armament@PRIMARY:
+		Name: primary
+		Weapon: ed_infantry_a01_machine_gun_vs_ground
+		MuzzlePalette:
+	Armament@SECONDARY:
+		Name: secondary
+		Weapon: ed_infantry_a01_machine_gun_vs_infantry
+		MuzzlePalette:
+	Armament@TERTIARY:
+		Name: tertiary
+		Weapon: ed_infantry_a01_grenade
+		MuzzlePalette:
+	AttackFrontal:
+		Armaments: primary, secondary, tertiary
+	EnhancedEncyclopedia:
+		Category: ED - Infantry
+		Order: 1
+		Animation: DATABASE/A01.FLC
+		Title: A01 Android
```

Use original name *Android A01*:

```suggestion
		Title: Android A01
```